src/components/vuestic-components/va-form/VaForm.vue

```diff
@@ -0,0 +1,162 @@
+<script>
+import { ContextPluginMixin, getContextPropValue } from '../../context-test/context-provide/ContextPlugin'
+
+const availableFormTags = ['VaInput', 'VaColorInput', 'VaDatePicker', 'VaCheckbox', 'VaRadio', 'VaSelect', 'VaFileUpload']
+const getAllChildren = (vm, children = []) => {
+  vm.$children.forEach(function (child) {
```

Comment:
This approach won't work. We have to stop as soon as we find form component.
We should not handle its internals.

I.e. you can have double input marked as form component. It handles validation and such internally.

Reply:
We need to know all child because we need to call reset method on them when a form will be clear

Reply:
@Kreezag No we don't.
If component handles its clear state - form should never interfere.
I.e. for double input I mentioned - default (reset) state could be: one input 0, second 100. And it should be provided from parent, given parent handles this logic.